### DISCUSSION TITLE
Add option to start Cloudlogcat when using cloudlogcat

### DISCUSSION THREAD

**Discussion by myhobby2** [magicbug/Cloudlog#3241] [Ideas / Feature Requests]
I sometimes forget to start cloudlogcat when working with cloudlog.. Can also be an option (I think easier), that the cloudlog is triggered after cloudlogcat is started (or at least an option that can be customized)

**Comment by magicbug** [comment]
Being that Cloudlog functions within the browser, I don't think there's any way of launching a desktop application.

You could of course make a custom shortcut on your computer but that would be the only way

**Reply by myhobby2** [reply]
And integrating this in cloudlogcat as an option? Then I'll use cloudlogcat to start my ham activities
